### PR TITLE
Add info on chats

### PR DESCRIPTION
**Pull request by lr2kate**

### CODE REVIEW

File: conversation_tree.textproto [hunk 1]
```diff
@@ -877,6 +878,192 @@ node {
...
+  link { name: "Региональные чаты в Швейцарии" }
+  link { name: "Региональные чаты в Европе" }
+}
+
+
+node {
+  name: "Региональные чаты в Швейцарии"
+  answer {
+    links {
+      text:""
```
Comment (crdev) [inline conversation_tree.textproto:893]: Do we always need text, even if it's empty? Also, please add a space after ':' if it is.

File: conversation_tree.textproto [hunk 2]
```diff
@@ -877,6 +878,192 @@ node {
...
+      url {
+        label: "Цюрих: https://t.me/zh_helps_UArefugee"
+        url: "https://t.me/zh_helps_UArefugee"
+      }
+      url {
+        label: "Цюрих: https://t.me/zh_helps_ukraine"
+        url: "https://t.me/zh_helps_ukraine"
+      }
+      url {
+        label: "Цюрих: https://t.me/Zh_helps_UA_mums (можно со всем вопросами про детей в Швейцарии скорее всего)"
```
Comment (crdev) [inline conversation_tree.textproto:903]: The label is too long (50 chars being an absolute maximum, as it won't get wrapped). There are also a few other violations down below.

Apologies, I've broken convo model tests, already sent out a PR to fix.

File: conversation_tree.textproto [hunk 3]
```diff
@@ -877,6 +878,192 @@ node {
...
+      url {
+        label: "Во (Лозанна), группа в фейсбуке"
+        url: "https://www.facebook.com/groups/3223556504546235/?ref=share"
+      }
+      url {
+        label: "Вале (Валлис): https://t.me/ValaisUkraine"
+        url: "https://t.me/ValaisUkraine"
+      }
+      url {
+        label: "Люцерн: https://chat.whatsapp.com/G8qFB8UzZINJJGPorMiCys"
```
Comment (crdev) [inline conversation_tree.textproto:971]: Please sort the button alphabetically by labels. This one starts a section of duplicates of lines 939+

File: conversation_tree.textproto [hunk 4]
```diff
@@ -877,6 +878,192 @@ node {
...
+  name: "Региональные чаты в Европе"
+  answer {
+    links {
+      text:""
+      url {
+        label: "Помощь украинцам в Дании 🇩🇰 "
+        url: "https://t.me/refugeesinDenmark"
+      }
+      url {
+        label: "Помощь украинцам в  Бельгии 🇧🇪 "
```
Comment (crdev) [inline conversation_tree.textproto:1036]: Duplicate whitespace: "в  Бельгии"

File: conversation_tree.textproto [hunk 5]
```diff
@@ -877,6 +878,192 @@ node {
...
+        url: "https://t.me/refugeesinBelgium"
+      }
+      url {
+        label: "Помощь украинцам в  Польше 🇵🇱 "
```
Comment (crdev) [inline conversation_tree.textproto:1040]: Duplicate whitespace: "в  Польше"

File: conversation_tree.textproto [hunk 6]
```diff
@@ -877,6 +878,192 @@ node {
...
+        url: "https://t.me/refugeesinPoland"
+      }
+      url {
+        label: "Помощь украинцам в Франции🇫🇷 "
```
Comment (crdev) [inline conversation_tree.textproto:1044]: typo: should be "во Франции". Please also add a whitespace before the flag.

File: conversation_tree.textproto [hunk 7]
```diff
@@ -877,6 +878,192 @@ node {
...
+      url {
+        label: "Помощь украинцам в Германии 🇩🇪 "
+        url: "https://t.me/refugees_in_Germany"
+      }
+      url {
+        label: "Помощь украинцам в Украине 🇺🇦 "
+        url: "https://t.me/refugeesinUkraine"
+      }
+      url {
+        label: "Помощь украинцам в Швеции🇸🇪 "
```
Comment (crdev) [inline conversation_tree.textproto:1056]: Please add a whitespace before the flag.

File: conversation_tree.textproto [hunk 8]
```diff
@@ -877,6 +878,192 @@ node {
...
+        url: "https://t.me/refugeesinSweden"
+      }
+      url {
+        label: "Помощь украинцам в Австрии🇦🇹 "
```
Comment (crdev) [inline conversation_tree.textproto:1060]: Please add a whitespace before the flag.

**Review by crdev — CHANGES_REQUESTED** [review]

File: conversation_tree.textproto (hunk 1 above)
Comment (skd) [reply conversation_tree.textproto]: Actually, without a text it should simply not work (no message would be sent, and hence no links with it).

**Review by skd — APPROVED** [review]

**Pull request merged**